src/test/java/org/folio/rest/GeneratePasswordRestLinkTest.java

```diff
@@ -45,6 +49,8 @@
 @RunWith(VertxUnitRunner.class)
 public class GeneratePasswordRestLinkTest {
 
+  private static final Logger log = LogManager.getLogger(GeneratePasswordRestLinkTest.class); 
```

Comment:
Why is a logger needed when there is no explicit logging in the class?

Reply:
I've noticed a number of files in the back-end where the logger is instantiated but not used-I assumed this was a convenience, so that if you needed to log something as part of debugging, the apparatus would be there.  So I didn't remove it from this instance.

Reply:
> I've noticed a number of files in the back-end where the logger is instantiated but not used-I assumed this was a convenience

Hmm, I imagine this is a mistake (likely either a copy / paste error or a failure to remove it). I don't think we need to introduce it prematurely here.

Reply:
Noted, removing it.